src/java.base/share/classes/java/lang/reflect/code/op/ExtendedOp.java

```diff
@@ -247,19 +247,35 @@ static BranchTarget getBranchTarget(CopyContext cc, Op op) {
         return null;
     }
 
+    static BranchTarget getBranchTarget(CopyContext cc, CodeElement<?, ?> codeElement) {
```

Comment:
You should be able to remove the op accepting set/getBranchTarget methods, since Op extends from CodeElement.